.github/workflows/conformance.yml

```diff
@@ -5,12 +5,14 @@ on:
     branches:
       - '**'
   pull_request:
+  # TODO: add cron
 
 jobs:
   conformance:
     strategy:
       matrix:
         java-version: [11, 17]
+        sigstore-env: [production, staging]
```

Comment:
At least in context of current latency issues you may want to avoid two staging jobs concurrently... Maybe worth considering if the two java versions give you value here considering ci.yaml already tests two versions?

Reply:
I can make them non concurrent

Reply:
there are security library differences between 11 and 17 that I want to potentially fail if it comes to it.

Reply:
These finish pretty quick so just made them run non-parallel